### ISSUE TITLE
Turning on Planner

### ISSUE THREAD

**Issue by N-d-w-32** [ndg63276/alexa-sky-hd#36]
The skill is working well and all the commands listed work like a charm! I have been able to activate the TV guide, but is there any way to access the planner and select from the list of recorded programmes? I have tried a number of different commands and don't seem to be having much luck with this. Thanks in advance for any help you can give.

**Comment by ndg63276** [comment]
No, it could open the planner I guess, but it has no idea what you have recorded. It's just a remote control.
